compiler/src/dmd/target.d

```diff
@@ -363,7 +363,7 @@ extern (C++) struct Target
     bool isX86_64;          // generate 64 bit code for x86_64; true by default for 64 bit dmd
     bool isX86;             // generate 32 bit Intel x86 code
     bool isLP64;            // pointers are 64 bits
-
+    extern(D) bool is64() const @safe { return isX86_64 /*|| isAArch64*/; }
```

Comment:
It looks like this is being turned into being synonymous with `isLP64` -> use that field instead and remove this function?

Reply:
With the rather confusing collection of nomenclature here, I am concerned that `is64` means "64 bit" and that implies that it includes `AArch64`. I'd rather stick with `X86_64` and `AArch64` explicitly, as those correspond to exact architecture names.

Reply:
Yeah, this broke DMD as a library backthen, and `isLP64` is now the way to get that. I would suggest name everything the same

Reply:
> It looks like this is being turned into being synonymous with isLP64 -> use that field instead and remove this function?

Done.